src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj

```diff
@@ -13,6 +13,7 @@
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
     <IsPackable>true</IsPackable>
     <IsShipping>true</IsShipping>
+    <!-- Nullable disabled because package targets netstandard2.0. -->
```

Comment:
We can enable nullable for netstandard projects
https://github.com/dotnet/aspnetcore/blob/b883b451b02876b9adaf3c4ce7ae0ac441f471bb/eng/targets/CSharp.Common.targets#L115

Reply:
Nice. TIL.